DummyProject/DummyProject/ViewController.swift

```diff
@@ -79,6 +79,62 @@ class ViewController: UIViewController {
         }
     }
 
+    // MARK: - Content (WebPage)
+
+    @IBOutlet weak var contentTextField: UITextField!
+    lazy var pickerView: UIPickerView = {
+        let pickerView = UIPickerView()
+        contentTextField.inputView = pickerView
+        return pickerView
+    }()
+
+    @IBAction func getWebPageButtonTapped(_ sender: Any) {
+        guard let pageType = WebPageType(value: contentTextField.text ?? "") else {
+            showAlertController(
+                title: "Content - Invalid Page Type",
+                message: "Unable to find this page type")
+            return
+        }
+        updateLoadingSpinnerVisibility(shouldShow: true)
+        RealifeTech.Content.getWebPage(forType: pageType) { [self] result in
+            switch result {
+            case .success(let webPage):
+                showAlertController(
+                    title: "Content",
```

Comment:
Do we need to localise these ?

Reply:
Probably not as it's in the dummy project which QA tests on.